### PR TITLE
#619 [Fix] Update numbers introduction

### PR DESCRIPTION
**Pull request by tesarect**
Updated numbers introduction.md file by including Assignment operator and Compound Assignment operators with examples.

This is my first contribution ever made. Please forgive me for inconsistent commit messages, description and titles so far. And please do correct me if the update wasn't up to the mark.

### CODE REVIEW

File: concepts/numbers/introduction.md [hunk 1]
```diff
@@ -41,4 +41,35 @@ When one of the involved types is a floating-point type, the result will also be
...
+
+```cpp
+            // Equivalent 
+x += 5;     // x = x + 5;
+x /= y;     // x = x / y;
+```
+In the first line, integer value `5` is added to existing value of variable `x` and is assigned back to `x`.
+similarly, value of `x` is divided by the value of `y` and is assigned back to `x`.
+
+There are other compound assignment operators which are related to bitwise operations.
```
Comment (vaeng) [inline concepts/numbers/introduction.md:72]: These are introduced later, so I would not yet talk about these.

File: concepts/numbers/introduction.md [hunk 2]
```diff
@@ -41,4 +41,35 @@ When one of the involved types is a floating-point type, the result will also be
 `5.0 / 2` and `5 / 2.0` will return `2.5`.
 `%` is the remainder operator and will return the remainder of an integer division: `5%3` is `2`.
 
+## Assignment operator
+
+The assignment operator assigns a variable with a literal.
+Always takes place from right to left, and never the other way around.
+
+```cpp
+int x, y;
```
Comment (vaeng) [inline concepts/numbers/introduction.md:50]: Please avoid uninitialized variables in Examples.
-> https://github.com/exercism/cpp/blob/main/concepts/basics/introduction.md

**Review by vaeng — COMMENTED** [review]
Thanks for your contribution. It is a good idea to talk about assignments as well.

- To keep up with changes in git it is useful to have every sentence on its own line: https://exercism.org/docs/building/markdown/markdown
- We will have a separate exercise for bitwise operations, so I would not mention them yet.
- If you use real words instead of `x` and `y`, the example is usually easier to follow. Like eggs and people:

```cpp
// we start with 0 people
int people{};
// we need 0 eggs
int eggs{}; 
// two people joined:
people += 2; 
// people is now 2
// let's add 3 eggs per person
eggs += 3 * people;
// eggs is now 6
```

File: concepts/numbers/introduction.md (hunk 1 above)
Comment (tesarect) [reply concepts/numbers/introduction.md:72]: Got it

**Comment by tesarect** [comment]
should you hit the `resolve conversation` button or should it be done by me ?

**Comment by tesarect** [comment]
@vaeng, after each commit should we make a new pull request each time for review ?

**Comment by vaeng** [comment]
> @vaeng, after each commit should we make a new pull request for review ?

You can re-request a review. But once I'm added as a reviewer, I get all the updates anyway. For me it's mostly a thing of being free and near a desktop to do a proper review

**Comment by tesarect** [comment]
Understood. So please do let me know if any more modifications needs to be done on the recent commit.

**Review by vaeng — APPROVED** [review]

**Comment by tesarect** [comment]
@vaeng, thanks for your time on the review. Whats the next step (anything that i need to do from my side) ?
and can i delete the branch ?

**Comment by vaeng** [comment]
Squash merge with a good commit message and delete the branch. I wanted you to the honors

**Comment by tesarect** [comment]
image

Is this the way(`open pull request`) i should merge ?

**Comment by tesarect** [comment]
@vaeng, I guess i messed up with multiple commits btw forked branch and main. All my earlier PR's was raised from my branch and i wasn't aware i could have done the merge directly from my branch. Instead, i squash and merge to forked main from branch. Sorry for the inconvenience(multiple PR's)

**Comment by vaeng** [comment]
No problem. It can be confusing and honestly I don't know what rights you have as a non-maintainer. I'll integrate your commit by the end of the week.

**Pull request closed without merging**